Addons/tsgExoticQuadrature.hpp

```diff
@@ -329,7 +329,7 @@ inline void shiftReferenceWeights(std::vector<double> const &vals, double shift,
  * Note that the values of the weight function will not be interpolated between the reference points.
  *
  * Examples of symmetric weight functions are \f$ \sin(x) \f$, \f$ \sin(x) / x \f$, constant weight, and
- * the Gauss-Chebyshev weights. On the other hand, \f$ \cos(x) \f$ is not symmetric with regards
+ * the Gauss-Chebyshev weights. On the other hand, \f$ \cos(x) \f$ is not symmetric with respect
```

Comment:
When we say `symmetric with respect to` it is something like the origin or the y-axis or some other point.

Howe about:
```
is not symmetric in the context of this algorithm
```

Reply:
Sounds good.

Changed.